bases/ecoindex/cli/arguments_handler.py

```diff
@@ -36,8 +37,13 @@ def get_urls_from_file(urls_file: FilePath) -> Set[str]:
 
 def get_urls_recursive(main_url: str) -> Set[str]:
     parsed_url = urlparse(main_url)
-    domain = parsed_url.netloc
-    main_url = f"{parsed_url.scheme}://{domain}"
+    netloc = parsed_url.netloc
+    domain = netloc
+    if os.environ.get('DOCKER_CONTAINER') == 'true':
```

Comment:
**🎯 suggestion(non-blocking):** Here, I think that you can add this to the `components/ecoindex/config/settings.py` file as bool with default value to `False`... This will help us in keeping consistency in our development

Reply:
Done